sources/packages/web/src/services/http/SupportingUserApi.ts

```diff
@@ -53,7 +54,18 @@ export class SupportingUserApi extends HttpBaseClient {
     applicationId: number,
   ): Promise<ApplicationSupportingUsersDTO[]> {
     return this.getCallTyped<ApplicationSupportingUsersDTO[]>(
-      `supporting-user/application/${applicationId}/aest`,
+      this.addHook(`supporting-user/application/${applicationId}`),
```

Comment:
I liked the idea. Can we call it addRoot, or addClientRoot?
I did not get the _hook_ in this context 😉